src/explore-education-statistics-admin/src/components/Page.tsx

```diff
@@ -41,6 +41,7 @@ const Page = ({ breadcrumbs = [], children, wide, pageTitle }: Props) => {
             id="main-content"
             role="main"
           >
+            {pageTitle && <h1 className="govuk-heading-xl">{pageTitle}</h1>}
```

Comment:
Does this have any impact on existing pages?

Reply:
Not anymore! Good spot